lib/ansible/plugins/connection/winrm.py

```diff
@@ -359,7 +359,7 @@ def put_file(self, in_path, out_path):
         if result.status_code != 0:
             raise AnsibleError(to_native(result.std_err))
 
-        put_output = json.loads(result.std_out)
+        put_output = json.loads(to_text(result.std_out))
```

Comment:
You might want to specify a different error handler here.  The default handler is surrogate_or_replace.  When non-utf8 byte sequences are encountered, it either uses surrogateescape if available or replace.  This can munge output on python2.  Since we're sending this to json.loads (rather than displaying it to the user) it might be better to use surrogate_or_strict as the error handler here.